link/src/main/java/com/stripe/android/link/ui/signup/SignUpViewModel.kt

```diff
@@ -103,8 +102,7 @@ internal class SignUpViewModel @Inject constructor(
         }.collectLatest { email ->
             delay(LOOKUP_DEBOUNCE)
             if (email != null) {
-                updateSignUpState(SignUpState.VerifyingEmail)
-                lookupConsumerEmail(email)
+                updateSignUpState(SignUpState.InputtingRemainingFields)
```

Comment:
why do we no longer need to look up the consumer email here?

Reply:
We discussed it [here](https://github.com/stripe/stripe-android/pull/9377/files#r1793882747). I was supposed to remove it as a follow up since we don't have persistent accounts anymore.